document/execution/instructions.rst

```diff
@@ -77,9 +89,9 @@ Numeric Instructions
 .. math::
    \begin{array}{lcl@{\qquad}l}
    (t\K{.}\CONST~c_1)~(t\K{.}\CONST~c_2)~t\K{.}\binop &\stepto& (t\K{.}\CONST~c)
```

Comment:
Should it not be (t\K{.}\CONST~c_2)~(t\K{.}\CONST~c_1) because the values on the stack are in inverse order?